### PR TITLE
Make string content and heredoc content visible

### PR DESCRIPTION
**Pull request by aibaars**
The contents of string literals and heredocs used to be hidden because the rule names for them start with an `_`.

@dcreager

### CODE REVIEW

File: src/grammar.json [hunk 1]
```diff
@@ -5035,7 +5035,7 @@
           },
           {
             "type": "PATTERN",
-            "value": "[^\\x00-\\x1F\\s:;`\"'@$#.,|^&<=>+\\-*/\\\\%?!~()\\[\\]{}]*"
+            "value": "[^\\x00-\\x1F\\s:;`\"'@$#.,|^&<=>+\\-*\\/\\\\%?!~()\\[\\]{}]*"
```
Comment (aibaars) [inline src/grammar.json:5038]: @maxbrunsfeld : I wonder where these changes come from. They seem to revert the changes of 7608c140f49168c7d97bb1ccdc3c2aeccc507245 . I ran `node node_modules/tree-sitter-cli/cli.js generate` to re-generate the files. Did  you use another version `tree-sitter` when you generated the files?

```
node node_modules/tree-sitter-cli/cli.js -V 
tree-sitter 0.17.3 (c439a676cf169e88234f768ca0f69d42e5bd68c5)
```

Comment (dcreager) [reply src/grammar.json:5038]: This looks innocuous, since it's just adding an escape to a forward slash.  I'm also curious where it came from.  I briefly tried bisecting this but couldn't find anything obvious.  The JavaScript standard says that [slashes should be escaped](https://tc39.es/ecma262/#sec-escaperegexppattern) (and if I'm reading it carefully, it furthermore says that slashes should _always_ be escaped).  But the current V8 source looks like it only escapes slashes that occur [outside of a character class](https://github.com/v8/v8/blob/4c2fd721d61ca944e0c4dd46295da214b861d02b/src/objects/js-regexp.cc#L387-L389).  This example is _inside_ a character class, so now I'm thoroughly confused.

Anyway, since it shouldn't affect the result, I'm :+1: to merge this.

**Review by dcreager — APPROVED** [review]

File: src/grammar.json (hunk 1 above)
Comment (dcreager) [reply src/grammar.json:5038]: A-ha!  Someone else has done the bisecting: nodejs/node#29888, v8/v8@17b9d8728ad4dd46ca36d834ec96c8e7a5914567

Slashes inside of character classes are _no longer escaped_ as of V8 7.7.206 and node.js 12.11.0. @aibaars can you double-check what version of node.js you have installed?

Comment (aibaars) [reply src/grammar.json]: I had version 10 installed. I'll upgrade and regenerate the files.

**Review by dcreager — APPROVED** [review]
